samples/sensor/mcp9808/README.rst

```diff
@@ -6,42 +6,65 @@ MCP9808 Temperature Sensor
 Overview
 ********
 
-Sample application that periodically reads temperature from the MCP9808 sensor.
+This sample application periodically (0.5 Hz) measures the ambient
```

Comment:
The doc changes in #21427 are preferred.

Reply:
Updates made (plus-minus, explanation of interleaved output).  Please continue documentation review here; #21427 will be rebased on top of this change when it goes in.